Dockerfile

```
@@ -43,7 +47,9 @@ RUN apt-get update \
         librabbitmq-dev \
         libxml2-dev libxslt-dev \
         locales \
-    && docker-php-ext-configure gd \
+        vim \
+        mycli \
+    && docker-php-ext-configure gd --with-jpeg --with-webp --with-freetype --with-xpm \
```

This is not synced with backend's one: is this on purpose? I don't mind enabling webp, but we don't need freetype nor xpm. Since php-api is aimed to be used in production, let's reduce the risk by not enabling things we know we don't use, what do you think?

---

Tu as raison, je n'étais pas sûr, mais c'est vite fait.
Et oui, il n'est pas sync à backend car backend se veut stable et mon php-api tjr pas adopté par tout le monde ;-) 

```suggestion
    && docker-php-ext-configure gd --with-jpeg --with-webp \
```